Directory.Packages.props

```diff
@@ -37,8 +37,8 @@
     <PackageVersion Include="Microsoft.AspNetCore.Components.Analyzers" Version="6.0.33" />
     <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="6.0.33" />
     <PackageVersion Include="Microsoft.AspNetCore.Components" Version="6.0.33" />
-    <PackageVersion Include="System.ServiceModel.Http" Version="6.2.0" />
-    <PackageVersion Include="System.ServiceModel.Primitives" Version="6.2.0" />
+    <PackageVersion Include="System.ServiceModel.Http" Version="8.0.0" />
```

Comment:
Disse må versjoneres iht til hvilket framework vi har.  Altså inn med conditionals

Reply:
@OsirisTerje Det er allerede gjort?

For .Net6: System.ServiceModel.Http 6.2.0
For .Net8: System.ServiceModel.Http 8.0.0
For .Net9: System.ServiceModel.Http 8.0.0 (9 har ikke blitt sluppet enda)

Eller misforstår jeg kommentaren din?

Virker som det er riktig at System.ServiceModel.Primitives kan fjernes.